src/ipyforcegraph/behaviors/_base.py

```diff
@@ -18,14 +18,6 @@ class BaseD3Force(Behavior):
     """A base for all ``d3-force-3d`` force wrappers."""
 
     _model_name: str = T.Unicode("BaseD3ForceModel").tag(sync=True)
-    key: str = T.Unicode(
```

Comment:
Was this intentional?

Reply:
indeed: this is now a conflict vs `main`

Reply:
That actually isn't used and should have been removed on gh-33. We use a different mechanism for "naming" forces e.g. the dictionary `GraphForces.forces`.